cmd/common.go

```diff
@@ -91,3 +98,20 @@ func ClusterInit(ctx context.Context, rkeConfig *v3.RancherKubernetesEngineConfi
 	}
 	return rkeState.WriteStateFile(ctx, stateFilePath)
 }
+
+func setS3OptionsFromCLI(c *cli.Context) *v3.S3BackupTarget {
```

Comment:
I think we still need to pass the access and secret keys here, if they are set in the command line, right ?

Reply:
yep, I have brought it back.